libmamba/tests/src/solver/test_problems_graph.cpp

```diff
@@ -534,6 +534,39 @@ namespace
         );
     }
 
+    auto create_sudoku(Context&, ChannelContext& channel_context)
+    {
+        auto db = solver::libsolv::Database{ channel_context.params() };
+        const auto xpt = db.add_repo_from_repodata_json(
+            mambatests::test_data_dir / "repodata/sudoku.json",
```

Comment:
I guess it is fine if we cannot support fetching remote files.

Reply:
This is a reasonably small (30'000l) file that I committed it to the repo. It's also nice to keep the test independent from the rest of Mamba.